DiscordChatExporter.Domain/DiscordChatExporter.Domain.csproj

```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="ByteSize" Version="2.0.0" />
```

Comment:
Sure, how'd you like it to work? Would it use the `TryParse(s, out result)` pattern, an enum pattern with success and failure objects (e.g. `FileSizeParseResult.Ok` and `FileSizeParseResult.Error`), or would you like a single FileSizeParseError class, where a null value indicates success, like how CliFx does it now? 

it might be simpler for you to do this yourself real quick once things are ready.